lib/tasks/importer.rake

```diff
@@ -1,20 +1,19 @@
-require_relative '../cms/hippo_import'
-
 namespace :cms do
   namespace :hippo do
-    desc 'Import Hippo XML data into database (options: FROM=folder_name TO=site_identifier DOCS=file_of_docs)'
-
-    task :import => :environment do
+    desc 'Import Hippo XML data into database \n(options: FROM=folder_name TO=site_identifier DOCS=file_of_docs TYPE=type)'
+    task import: :environment do
       from  = ENV['FROM']
       to    = ENV['TO']
       docs  = ENV['DOCS']
+      type  = ENV['type'] || 'base'
 
       puts "Importing CMS Data from file [#{from}]..."
 
       file = File.read(File.expand_path(from))
       docs = File.read(File.expand_path(docs)).split("\n") if docs
 
-      imported = HippoImport.new(file, docs, to).import!
+      import = "Cms::HippoImporter::#{type.classify}".constantize.new(data: file, docs: docs, to: to)
+      imported = import.import!
```

Comment:
with this you can run the corporate pages with:

`rake cms:hippo:import TYPE=corporate_page ...`